omod/src/main/resources/apps/aijar_homepage_app.json

```diff
@@ -53,7 +54,7 @@
         "url": "cohortBuilder.list",
         "icon": "icon-building",
         "order": 105,
-        "requiredPrivilege": "Manage Reports"
+        "requiredPrivilege": "App: aijar.manageReports"
```

Comment:
@slubwama Better to add permissions to reportingui that can be leveraged

Reply:
@ssmusoke That means the app it self goes away from aijar to the reporting compatibility

Reply:
The app can be in our custom module, but the privileges have to come from the respective modules, reportingcompatibility in the case of the cohort builder. 

Remember the approach is UgandaEMR only has configuration and customization, all other functionality belongs to the respective modules

Reply:
@ssmusoke This is an app access privilege not the restriction on the objects of Cohort Builder. All I am doing is adding permissions for all apps that are being overridden  in our implementation.